lib/src/tests/math.rs

```diff
@@ -0,0 +1,50 @@
+use super::*;
+
+make_test!(mathblock1,
```

Comment:
for all the test you add, you need to run 
1. `cargo insta test`
2. `cargo insta review` and accept the result (alternatively you can use the other macro which should be named something like test_eq)